app/javascript/pages/country/widgets/widget-tree-cover-gain/widget-tree-cover-gain.js

```diff
@@ -56,48 +85,36 @@ class WidgetTreeCoverGainContainer extends PureComponent {
       item => item.value === settings.indicator
     );
     const regionPhrase =
-      settings.indicator === 'gadm28_only'
+      settings.indicator === 'gadm28'
```

Comment:
would it make sense to keep a list of constants with the indicators names? to avoid many of these?

Reply:
They are currently in a constants, but this is the default. I will create a default location one and add on the next PR. Great idea.